recipes/define-symbols.md

```diff
@@ -0,0 +1,28 @@
+# Define symbols
+
+## Problem
+
+I want to define several variables that have their symbol as value.
+For example `(make-symbols x y z)` should define `x`, `y` and `z`
+which evaluate to `x`, `y` and `z` respectively.
+
+## Solution
+
+```scheme
+(define-syntax make-symbols
+  (syntax-rules ()
+    ((_ symbol ...)
+     (begin
+       (define symbol 'symbol) ...))))
```

Comment:
Missing the recursive call to `make-symbols`?

Reply:
Why would a recursive call be necessary?

Reply:
Wow, I didn't know about the `syntax-rules` feature you're using.

This one?

> Pattern variables that occur in subpatterns followed by one or more instances of the identifier ellipsis are allowed only in subtemplates that are followed by as many instances of ellipsis. They are replaced in the output by all of the elements they match in the input, distributed as indicated.